### ISSUE TITLE
git repositories "keyed with a key generation that we don't know"

### ISSUE THREAD

**Issue by lluchs** [keybase/keybase-issues#3332]
Any action with git repositories in keybase prints the following error:

```
The data for 6d9554f1c234355ebbab69af8c6c3a16 is keyed with a key generation (2) that we don't know
```

I don't think I did anything special. I'm using these git repositories from a single computer only.

I'm using keybase from the Arch Linux repositories.

**Comment by strib** [comment]
@lluchs please do a `keybase log send` and post the resulting log ID here.  I'll take a look.

**Comment by lluchs** [comment]
Thanks! Here's the log ID: be281b0a867655a535d2901c

**Comment by strib** [comment]
You are experiencing a bug that hit a few other people, as we migrate folders from an old keying behavior to a new, better one.  There will be a fix in the next Keybase release, but I prepared a Linux build for you that includes the fix, if you want to try it out: 

```sh
dpkg -i /keybase/private/strib,luchs/keybase-2.13.1-20190121180822.47b0808949-amd64.deb
```
Once you access the folder and write to it with this build, it should be fixed for any other devices as well.

Let me know how it goes, or if you have other questions or concerns.

**Comment by strib** [comment]
(For the record: the above release is the same as our current release, but including the fix of keybase/client#15606.)

**Comment by lluchs** [comment]
Awesome, that did indeed fix the issue. Thanks a lot!
